cmd/starter/c/starter.c

```diff
@@ -1417,8 +1472,8 @@ __attribute__((constructor)) static void init(void) {
              * escalation from master process, because container network requires
              * privileges
              */
-            if ( sconfig->starter.isSuid  ) {
-                priv_drop(false);
+            if ( sconfig->starter.isSuid && geteuid() == 0 ) {
```

Comment:
Please could you update the comment above on 1471 to reflect that we are not dropping priv temporarily in the same way.. but instead we are setting capabilities instead

Reply:
Will do thanks !